### DISCUSSION TITLE
Provide a suggested donation amount

### DISCUSSION THREAD

**Discussion by jdgemm** [davetapley/helium-tax#15] [Ideas]
Works well. To help users donate provide a suggested amount (e.g., $5 HNT/Hotspot) or whatever you think is fair.

**Comment by davetapley** [comment]
Good idea, thanks! (3873805aaa274e370e0774bd1c43d4431be5dbe7)

**Reply by davetapley** [reply]
Ended up removing this because price is too volatile to me meaningful 🤯
